### PR TITLE
README: add build instruction and github release badge

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What problem does this PR solve? 

- add build instruction, ref #26 
- add release badge

### What is changed and how it works?

- update the README doc

### Check List 

Tests 

 - No code

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@GregoryIan @amyangfei PTAL

**Comment by GregoryIan** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -15,6 +16,25 @@
...
+If you only want to build binaries, you can run:
+```bash
+make dm-worker  # build DM-worker
+
+make dm-master  # build DM-master
+
+make dmctl      # build dmctl
+``` 
+
+When DM are built successfully, you can find binaries in the `bin` directory.
```
Comment (amyangfei) [inline README.md]:
```suggestion
When DM is built successfully, you can find binaries in the `bin` directory.
```
`is` or `are`, which is better

Comment (csuzhangxc) [reply README.md]: I chose `is`.

**Pull request merged**